clueless/settings.py

```diff
@@ -118,3 +117,5 @@
 # https://docs.djangoproject.com/en/1.10/howto/static-files/
 
 STATIC_URL = '/static/'
+
+from .settings_secret import *
```

Comment:
Really strange.  Tried to edit in windows and it doesn't even show up.  If I delete that line and recreate it in the web editor it doesn't even allow me to commit because it doesn't think there is a change...